packages/backend-common/src/compat/legacy/legacy.ts

```diff
@@ -87,7 +152,12 @@ export function makeLegacyPlugin<
   return (
     name: string,
     createRouterImport: Promise<{
-      default: LegacyCreateRouter<TransformedEnv<TEnv, TEnvTransforms>>;
+      default: LegacyCreateRouter<
+        TransformedEnv<TEnv, TEnvTransforms> & {
+          tokenManager: ServerTokenManager;
+          identity: LegacyIdentityService;
```

Comment:
maybe this is obvious, but what made these necessary to move outside of the TransformedEnv?

Reply:
Since they're not using service refs you can't apply any transforms to them either. These just end up being extra values available on the env